### PR TITLE
Refactor and move System Info to elementor\modules

### PR DESCRIPTION
**Pull request by udidol**
## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [X] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Description
An explanation of what is done in this PR

* Moved Elementor's `System Info` report module into `elementor\modules` and refactored it as a module

## Quality assurance

- [X] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

### CODE REVIEW

File: modules/system-info/module.php [hunk 1]
```diff
@@ -1,53 +1,55 @@
...
 }
 
 /**
- * Elementor main system info page.
+ * Elementor system info module.
  *
- * Elementor main system info page handler class responsible for creating system info
- * reports and displaying them on WordPress dashboard under Elementor setting.
+ * Elementor system info module handler class is responsible for registering and
+ * managing Elementor system info modules.
```
Comment (kobizz) [inline modules/system-info/module.php]:
```suggestion
 * managing Elementor system info reports.
```

File: modules/system-info/module.php [hunk 2]
```diff
@@ -67,78 +69,85 @@ class Main {
...
 
-		if ( ! ( $reporter instanceof Base_Reporter ) ) {
-			return new \WP_Error( 'Each reporter must to be an instance or sub-instance of `Base_Reporter` class.' );
-		}
+		$settings['reporter_properties'] = $reporter_properties;
 
-		if ( ! $reporter->is_enabled() ) {
-			return false;
-		}
+		$settings['namespaces'] = [
```
Comment (kobizz) [inline modules/system-info/module.php]: Are we still need these settings?

File: modules/system-info/module.php [hunk 3]
```diff
@@ -393,7 +349,7 @@ private function init_settings() {
 	 *
 	 * @return mixed
 	 */
-	final public function get_settings( $setting = null, array $container = null ) {
+	/*final public function get_settings( $setting = null, array $container = null ) {
```
Comment (kobizz) [inline modules/system-info/module.php]: Remove this method

**Review by kobizz — CHANGES_REQUESTED** [review]

File: modules/system-info/module.php (hunk 2 above)
Comment (udidol) [reply modules/system-info/module.php]: No, will delete

**Review by kobizz — APPROVED** [review]

**Pull request merged**